src/metabase/driver/generic_sql.clj

```diff
@@ -430,9 +376,7 @@
 (defn analyze-table
   "Default implementation of `analyze-table` for SQL drivers."
   [driver table new-table-ids]
-  ((analyze/make-analyze-table driver
-     :field-avg-length-fn   (partial field-avg-length driver)
-     :field-percent-urls-fn (partial field-percent-urls driver))
+  ((cached-values/make-field-extractor driver)
```

Comment:
other insanity... calling a function that returns a function that is then used to get the field values. Instead of just making that function top-level and calling it directly. Luckily, this code is only copy-pasted 4 times in the code base, and only one of them is passing the `:calculate-row-count? false` options that aren't used *anywhere*.